### DISCUSSION TITLE
노트북화면으로 화상회으를 하는데 상대방 얼굴이 잘 안보여요

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#61] [제안하기]
화상회의 할 때 상대방 얼굴이 잘 안보여서 스크롤을 통해 확대하니 사용자 얼굴이 같이 작아져서 확인하기 어렵네요 ㅠ

**Comment by prism27** [comment]
안녕하세요! TJ 입니다! 답변이 늦었습니다.

어?  저희가 스크롤을 해도 설정한 크기를 어느정도 유지하는 상황이었는데,
기능 개선 과정에서 변경점이나 Side Effect 가 있는 지 확인해 보겠습니다.

제보 감사합니다!!

**Comment by prism27** [comment]
아 제가 질문 이해를 잘못한 것 같네요 ㅠㅠ

스크롤로 확대를 했는데 얼굴은 오히려 작아지는 상황이 불편하신 것 같습니다 (아니라면 다시 커멘트좀..;;)
사실 이전 댓글에서 말씀드린 것과 같이 스크롤에 따라 변경되는 크기를 최소화 하고자 하는 컨셉으로 잡았는데,
크게 보고싶은 Needs 에 대해선 커버가 되지 않는 것 같습니다. 

이 부분도 좀 더 최적의 비율을 찾아 조정하고, 확대했을 땐 좀 더 큰 비디오 영역을 제공하는 등 개선을 고민해 보도록 하겠습니다.

감사합니다!!

**Reply by act-lamos** [reply]
네 맞아요 변경 후에 스크롤을 해도 사용자 얼굴이 작아져서 크게 보기 힘들었습니다 ㅎㅎ
그래서 격자뷰로 갔는데 거기는 또 전체사용자가 안나오고, 화자가 다음 페이지에 있고 한 불편함이...
